### PR TITLE
Feature - custom port forwarding documentation

### PR DESCRIPTION
**Pull request by zhenik**
Offer https://github.com/fredrikhgrelland/vagrant-hashistack/issues/343#issuecomment-704882273
Option 2

### CODE REVIEW

**Review by claesgill — APPROVED** [review]
LGTM

**Review by dangernil — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -349,6 +350,42 @@ To see a full example of how to start a new project based on this box go to [tem
...
+           [-Q query_option] [-R address] [-S ctl_path] [-W host:port]
+           [-w local_tun[:remote_tun]] destination [command]
+```
+
+Example shows how to forward minio(which uses as artifactory) port `:9000` from vagrant box to localhost port `:6666`
+```text
+vagrant ssh -- -L 6666:localhost:9000
+```
+
+> Syntax `vagrant ssh -- -L <local machine port>:<host inside machine>:<port inside machine>`
```
Comment (pdmthorsrud) [inline README.md:373]:
```suggestion
The syntax is

```text
Syntax `vagrant ssh -- -L <local machine port>:<host inside machine>:<port inside machine>`
```

and the example below shows how to forward MinIO (which is used as an artifact manager) port `:9000` from vagrant box to localhost port `:6666`

```text
vagrant ssh -- -L 6666:localhost:9000
```
```

File: README.md [hunk 2]
```diff
@@ -349,6 +350,42 @@ To see a full example of how to start a new project based on this box go to [tem
...
+Example
+```text
+vagrant ssh -- -L 7001:localhost:9999 -L 7002:localhost:9000
+```
+
+
+
+
+
+
```
Comment (pdmthorsrud) [inline README.md:388]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (pdmthorsrud) [reply README.md:373]: Uuuh, not sure what happened with this suggestion?

**Review by claesgill — APPROVED** [review]
Still LGTM

**Review by pdmthorsrud — APPROVED** [review]

**Comment by pdmthorsrud** [comment]
Waiting for tests, then merging this.

Make sure to merge before #389, done some small changes to TOC in that one, which would conflict

**Pull request merged**